### PR TITLE
please check

### PR DESCRIPTION
**Pull request by jhaycawasa00**

### CODE REVIEW

**Comment by kchia** [comment]
@jhaycawasa00 can you take a screenshot of the page and attach it to your pr?

File: style.css [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+
+
```
Comment (kchia) [inline style.css]: Can you remove these empty spaces?

**Comment by kchia** [comment]
@jhaycawasa00 you can add the screenshot as a comment here.

**Comment by jhaycawasa00** [comment]
> @jhaycawasa00 you can add the screenshot as a comment here.

Screen Shot 2019-08-14 at 2 57 56 PM

**Review by kchia — APPROVED** [review]

**Pull request merged**